packages/ember-data/lib/system/model/errors.js

```diff
@@ -91,19 +91,8 @@ var MapWithDefault = Ember.MapWithDefault;
   @uses Ember.Enumerable
   @uses Ember.Evented
  */
-export default Ember.ArrayProxy.extend(Ember.Evented, {
-  /**
-    Register with target handler
+export default Ember.ArrayProxy.extend({
 
-    @method registerHandlers
-    @param {Object} target
-    @param {Function} becameInvalid
-    @param {Function} becameValid
-  */
-  registerHandlers: function(target, becameInvalid, becameValid) {
-    this.on('becameInvalid', target, becameInvalid);
-    this.on('becameValid', target, becameValid);
-  },
```

Comment:
I think @pangratz is correct. @arenoir do you mind adding `registerHandlers` back with a deprecation warning and continuing to use the `Ember.Evented` mixin even though the behavior is no longer needed?